clients/drcachesim/tools/filter/record_filter.cpp

```diff
@@ -95,7 +96,9 @@ parse_string(const std::string &s, char sep = ',')
     std::vector<T> vec;
     do {
         pos = s.find(sep, at);
-        unsigned long long parsed_number = std::stoull(s.substr(at, pos));
+        // base = 0 allows to handle both decimal and hex numbers.
+        unsigned long long parsed_number =
+            std::stoull(s.substr(at, pos), nullptr, /*base = */ 0);
```

Comment:
Note this change.
I don't think it has any adverse effect on the parsing of other options, but worth mentioning.